src/views/OfflineDatalog.vue

```diff
@@ -91,15 +138,26 @@ export default {
       this.selectedChannel = payload;
       let nRecords = 0;
 
+      //? Apply user datetime offset to series data.
+      let renderdata = structuredClone(
+        this.datalogRecords[this.selectedChannel["name"]]
+      ); //Deep Copy
+      renderdata.forEach((field) => {
+        for (let i = 0; i < field["data"].length; i++) {
+          field["data"][i][0] += this.dateTimeOffset;
+        }
+        return field["data"];
+      });
```

Comment:
is this the same method with `onSelectedDate` ?

Reply:
Yep it's same method with onSelectedDate